### DISCUSSION TITLE
Stone Haven concept

### DISCUSSION THREAD

**Discussion by Nimr1** [sunflower-land/sunflower-land#2294] [Ideas]
Stone Haven Mine walls to unlock crop points where you can plant special crops. Walls reset each week.

New Crops
•	Mushroom
•	Rhubarb
•	White Asparagus
•	Onion
New Resources
•	Coal: melt ores 
•	Salt: boost food XP by 10% 1 time use
New Ores
•	Emerald ore
•	Ruby ore
•	Diamond ore
•	Gold ore
•	Silver ore
•	Iron ore
•	Stone ore
Miner Bears Get your hands on the latest addition to the game, the Miner Bears.
Buildings
•	Forge: smelt ores and sell them 
•	Jewelry station: craft bumpkin wearables
Missions
•	Weekly mission: Hunt for ores. Each week a set amount of Emerald, Ruby, and Diamond is minable.
•	Wearables: Complete four tasks to earn wearables with special bonuses.
1.	Mine 10 walls
2.	Smelt 5 ores = Miner hat with +10% ore yield
3.	Harvest 200 Red Onions = Gloves.
4.	Collect 1 Ruby/Emerald = hammer.
Achievements Complete the following achievements to earn rewards:
•	No time to sleep: Harvest 100 dark crops. +5sfl
•	My precious: Find a Ruby/Emerald. +3 gold.
•	A girl's best friend: Collect 10 diamonds. Miner bear.
Goblin Village Visit the Goblin Village to purchase unique items:
•	Silver necklace: Increase diamond chance by 20% and emerald/ruby by 5%
•	Mine cart: Increase daily ores limit
•	Ancient anvil: -10% chance for 2.5 ingots
•	Obsidian pickaxe: Can mine stone free
Helios Get your hands on the latest equipment to help you harvest resources more efficiently:
•	Coal burner: Boost coal collected by 50%
•	Salt grinder: Salt use cost -30%
•	Heat lamp: Dark crops harvest speed +20%
Auction Bid for special items in the auction:
•	Shoes
•	Shirt
•	Glasses
Stone Haven Mine walls to unlock crop points where you can plant special crops. Walls reset each week.
•	Rent NPC miners to mine for you
Bumpkin Miner Get access to special items by mining rocks:
1.	Red onion soup + silver pickaxe + golden drill = 1 diamond, breaks in a square
2.	Roasted asparagus + silver pickaxe = 10% chance of Emerald, 30% gold, 40% salt + coal, 20% diamond, breaks 3 blocks in a row
3.	Rhubarb salad + golden drill = 10% chance of Ruby, 20% diamond, 40% salt + coal, 30% silver, breaks 3 blocks in a row

Goblin Miner 
Option 1: Use a pickaxe and any smoothie shack meal  (Rotation weekly ) to receive 1 stone, 0.25 silver, and 1 coal.
Option 2: Use a stone pickaxe and any kitchen meal (Rotation weekly )  to receive 1 iron, 0.5 silver, 0.5 coal, and 0.5 salt. This option also allows for breaking walls at a 50% chance.
Option 3: Use an iron pickaxe and deli meal (Rotation weekly ) to receive 1 gold, 1 silver, and 1 salt. This option guarantees breaking walls and provides valuable resources.
Foods Get access to special foods that are sellable and eatable:
•	Rhubarb salad
•	Roasted asparagus
•	Onion soup
•	Vegetarian burger
SFL Land Use silver for land expansion and craft special decorations:
•	Diamond decorations
•	Ruby decorations
•	Emerald decorations

Jewelry station crafts bumpkin wearables: 
Silver arm band Silver  headpiece Golden chain Golden ring Emerald ring Ruby ring Emerald necklace Ruby necklace Diamond necklace Diamond ring

Forge uses coal to Forge ingots to sell at base price or use at Jewelry station Ingots

**Comment by Nimr1** [comment]
This is just a short concept of Stone Haven island. No images as 1- I suck in art and 2- its just an idea  for now

**Reply by rasheed19871** [reply]
that is very interesting idea, even if they don't implement the whole new item things i hope from this proposal they get some item ingame time to time.

**Reply by Nimr1** [reply]
I definitely think we should have more Helios Items everyone can earn as the general casual player can only get those items if they stay true to their name . The term "Casual farm game " makes it ironic for Goblin blacksmith items as they are First come first serve and very grind intensive.  I think there is separate proposal which i love with the wheel barrow and bale etc all  great for casual players. 

What is your thoughts on they rework the NFT's to unlimited supply but time limited? Think there are lots of Benefits to the players from this angle.
